src/mono/wasm/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="BuildApp">
+  <PropertyGroup>
```

Comment:
Hmm, I did this and it runs locally, but it can't find Microsoft.Build.NoTargets on the helix machines. Is there something I need to do to get it there?